calico/datadog_checks/calico/check.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any
+
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheck
+
+class CalicoCheck(OpenMetricsBaseCheck):
```

Comment:
```suggestion
class CalicoCheck(OpenMetricsBaseCheckV2):
```

This new implementation is simpler to configure and more performant, the downside is that is Py3 only